docker/images/birt/run.sh

```diff
@@ -4,5 +4,9 @@
 aws s3 sync s3://promed-database/ ./
 mongorestore -h 172.30.2.160 -d promed -c blindspots ./dump/promed/blindspots.bson
 
+#Run the annual sightings script
+cd /birt-meteor/tools &&\
+npm run-script annualSightings
```

Comment:
Good catch! I just updated to use the -u flag with the $MONGO_URL var set in the container file:
https://github.com/ecohealthalliance/infrastructure/blob/master/docker/containers/birt.yml#L5